dali/pipeline/operators/python_function/dltensor_function.cc

```diff
@@ -19,13 +19,25 @@ namespace dali {
 
 DALI_SCHEMA(DLTensorPythonFunctionImpl)
     .AddParent("PythonFunctionImplBase")
+    .AddOptionalArg("synchronize_stream", "", true)
     .NumInput(0, 256)
     .OutputFn([](const OpSpec &spec) {return spec.GetArgument<int>("num_outputs");})
     .MakeInternal();
 
 DALI_SCHEMA(DLTensorPythonFunction)
     .AddParent("PythonFunctionBase")
-    .DocStr("Execute a python function that operates on DLPack tensors.")
+    .AddOptionalArg("synchronize_stream",
+        R"code(Make DALI synchronize its CUDA stream before calling the python function.
+Should be set to false only if the called function schedules the device job
+to the stream used by DALI.)code", true)
+    .DocStr(R"code(Execute a python function that operates on DLPack tensors.
```

Comment:
To be consistent with other schema definitions, could you move the DocStr to line 28?